prodtools/form.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, tk_root, default_xml_path, is_converter_enabled, function):
 
         self.is_converter_enabled = is_converter_enabled
         self.generate_pmc_package = False
+        self.optimise_images_for_web = False
```

Comment:
Não invalida, mas é importante alterar em algum momento trocar **optimise** por **optimize**!

Reply:
https://painintheenglish.com/case/920

Reply:
@jamilatta "optimise" é inglês Britânico, "optimize" é inglês americano. Ocorre em outras palavras: como "center" e "centre".

Reply:
Eu sei, só precisamos decidir qual dialeto iremos seguir como padrão, se for o britânico tudo bem!